sqs/user_template.go

```diff
@@ -27,13 +24,12 @@ const (
 type AccessPolicy = string
 
 const (
-    AccessPolicyFull AccessPolicy = "full"
-    AccessPolicyProducer AccessPolicy = "producer"
-    AccessPolicyConsumer AccessPolicy = "consumer"
+	AccessPolicyFull     AccessPolicy = "full"
+	AccessPolicyProducer AccessPolicy = "producer"
+	AccessPolicyConsumer AccessPolicy = "consumer"
```

Comment:
Ahyes, I think I had copypasta'd this from a github  comment.

Reply:
this is my editor doing `go fmt` (well, `goimports` really) on save. I wasn't even aware it had changed things here.

Reply:
I have just discovered a wonderful way of disguising my `git blame` then.

Reply:
@risicle nice try but I don't use blame, I use [git pickaxe](http://www.philandstuff.com/2014/02/09/git-pickaxe.html) 😄

Reply:
Interesting - I think the linked situation is an example I'd definitely advocate in favour of inline comments for. Mostly because most developers _just won't_ dig through the commit messages like that and that git blame gets muddied by people playing games with linters.

But this is a topic I could talk about at extreme length - I feel quite passionate about inline comments.

Reply:
I feel passionate that `git blame` is a bad tool and people shouldn't use it, and it's a shame it has so much more mindshare than pickaxe

Reply:
I can think of lots of cases though where pickaxe will not give you what quite you want or subtly hide commits which are ever so slightly outside your search term...

Reply:
I have used pickaxe consistently for years (check the blog post date) and it has generally been really good at finding the things I need. If I don't find the commit I want immediately, it's usually pretty obvious and I can tweak the search term as needed.